Dockerfiles/debug.Dockerfile

```
@@ -0,0 +1,61 @@
+# Build the manager binary
+ARG GOLANG_VERSION
+ARG USE_LOCAL=false
+ARG OVERWRITE_MANIFESTS=""
+
+################################################################################
+FROM registry.access.redhat.com/ubi8/go-toolset:$GOLANG_VERSION as builder_local_false
+ARG OVERWRITE_MANIFESTS
+# Get all manifests from remote git repo to builder_local_false by script
+USER root
+WORKDIR /opt
+COPY get_all_manifests.sh get_all_manifests.sh
+RUN ./get_all_manifests.sh ${OVERWRITE_MANIFESTS}
+
+################################################################################
+FROM registry.access.redhat.com/ubi8/go-toolset:$GOLANG_VERSION as builder_local_true
+# Get all manifests from local to builder_local_true
+USER root
+WORKDIR /opt
+# copy local manifests to build
+COPY odh-manifests/ /opt/odh-manifests/
+
+################################################################################
+FROM builder_local_${USE_LOCAL} as builder
+USER root
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY apis/ apis/
+COPY components/ components/
+COPY controllers/ controllers/
+COPY main.go main.go
+COPY pkg/ pkg/
+COPY infrastructure/ infrastructure/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -gcflags "-N -l" -a -o manager main.go
+RUN GOOS=linux GOARCH=amd64 GOPATH=/usr go install github.com/go-delve/delve/cmd/dlv@latest
```

```suggestion
RUN GOBIN=/usr/bin go install github.com/go-delve/delve/cmd/dlv@latest
```
You should add --platform in make build cmd as we had discussion.